Groups/GroupFinder.ascx.cs

```diff
@@ -1959,8 +1960,15 @@ private void ShowResults()
                     {
                         if ( !string.IsNullOrWhiteSpace( tbPostalCode.Text ) )
                         {
-                            mapCoordinate = new LocationService( rockContext )
-                                .GetMapCoordinateFromPostalCode( tbPostalCode.Text );
+                            try
+                            {
+                                mapCoordinate = new LocationService( rockContext )
+                                    .GetMapCoordinateFromPostalCode( tbPostalCode.Text );
+                            }
+                            catch
+                            {
+                                ShowWarning( $"An invalid {tbPostalCode.Label} was provided. If {tbPostalCode.Label} is required it will be reset to the default location." );
```

Comment:
```suggestion
                                ShowWarning( $"{tbPostalCode.Text} is an invalid {tbPostalCode.Label}. If {tbPostalCode.Label} is required, it will be reset to the default location." );
```